src/components/checkbox/__tests__/Checkbox-test.tsx

```diff
@@ -103,7 +103,7 @@ it('should have a error style when not checked', async () => {
 
 it('should be keyboard focusable while disabled', async () => {
   const onCheck = jest.fn();
-  const { user } = setupCheckbox({ label: 'me', checked: false, isDisabled: true, onCheck });
+  const { user } = setupCheckbox({ ariaLabel: 'me', checked: false, isDisabled: true, onCheck });
```

Comment:
Why are those `label` -> `ariaLabel` changes necessary?

Reply:
Ah because of the changes in the setup function?

Reply:
right, I think there was a typo